.github/workflows/ci-sdxl.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - shortfin - SDXL
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/ci_linux_x64-libshortfin.yml'
+      - 'shortfin/**'
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/ci_linux_x64-libshortfin.yml'
+      - 'shortfin/**'
+
+permissions:
+  contents: read
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+env:
+  IREE_REPO_DIR: ${{ github.workspace }}/iree
+  LIBSHORTFIN_DIR: ${{ github.workspace }}/shortfin/
+
+jobs:
+  build-and-test:
+    name: Build and test
+    runs-on: ubuntu-24.04
```

Comment:
@saienduri What will it take to get this .yml spinning on a MI300x?

Reply:
I noticed some other jobs running on a llama MI300 runner, can we do the same for SDXL?

Reply:
Yeah we can. Do we want SPX? Also how long would a job take (seems pretty quick) and how often would it run?

Reply:
SPX is ok to start, we can parametrize a bit once we get larger scenario tests/benchmarks running nightly. The e2e test here should take under a minute to complete, but we will want a place with room to grow. maybe a small suite (splat weights, small scenarios) for pre-submit and a slow suite (like mlperf CI) that runs nightly?

Reply:
Ok cool, free to get started with this label: `mi300-sdxl-kernel`. This is shared with the kernel benchmarking repo, but other than a nightly there isn't too much traffic there. When we are ready for a nightly here, we can find a good time to schedule it